_posts/en/newsletters/2020-07-29-newsletter.md

```diff
@@ -0,0 +1,236 @@
+
+- [What are "leaf versions" in Taproot?]({{bse}}97104)
+  Michael Folkson and Pieter Wuille explain that each leaf node of a
+  [taproot][topic taproot] tree commits to both a leaf version and a script.
+  This leaf version can be used for upgradability and designates the script
+  semantics that apply just to that leaf.
+
+- [What are the different upgradability features in the BIP-Taproot (BIP341) proposal?]({{bse}}96951)
+  Michael Folkson answers a question from Twitter regarding the different ways
+  in which taproot can enable upgradability including leaf versions for script
+  semantics, repurposing opcodes for future functionality, pubkey types, and the
+  annex for new fields.
+
+- [Is there an active list of BIPs currently open?]({{bse}}97043)
+  Pieter Wuille describes various methods that have previously been used to
+  activate soft forks and notes that, while there are currently no unactivated
+  soft forks in Bitcoin Core, there is a discussion about the activation method
+  for taproot. Pieter also answers a [similar question][stack exchange miner
+  signaling] about mining signaling support.
+
+- [Could we skip the Taproot soft fork and instead and use Simplicity to write the equivalent of Taproot scripts?]({{bse}}97049)
+  Michael Folkson, quoting Pieter Wuille, outlines the current state of
+  [Simplicity][news96 simplicity] noting also that integrating Simplicity in
+  taproot as a leaf version would be preferable.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.9.0rc3][C-Lightning 0.9.0] is a release candidate for
+  an upcoming major release.  It adds support for the updated `pay`
+  command and new `keysend` RPC described in [last week's
+  newsletter's][news107 notable] *notable code changes* section.  It also
+  includes several other notable changes and multiple bug fixes.
+
+- [Bitcoin Core 0.20.1rc1][Bitcoin Core 0.20.1] is a release candidate
+  for an upcoming maintenance release.  In addition to bug fixes and
+  some RPC behavior changes resulting from those fixes, the planned
+  release provides compatibility with recent versions of [HWI][topic
+  HWI] and its support for hardware wallet firmware released for the
+  [fee overpayment attack][].
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19473][] adds support for `networkactive` as both a command line
+  start-up and configuration file option. Setting this option enables or
```

Comment:
nit: not sure "support for", "both" and "start-up" are needed; maybe just "add ... as a command line and configuration file option"